packages/netsuite-adapter/src/client/sdf_client.ts

```diff
@@ -457,6 +473,16 @@ export default class SdfClient {
       executor,
       timeoutInMinutes
     )
+    const importResult = actionResult.data as ImportObjectsResult
+    const failedTypeToInstances = new collections.map.DefaultMap<string, string[]>(() => [])
+
+    importResult.failedImports.forEach(failedImport => {
+      if (failedImport.customObject.result.message.includes('unexpected error')) {
+        log.debug('Failed to fetch (%s) instance due to SDF unexpected error: (%s)', failedImport.customObject.type, failedImport.customObject.id)
+        failedTypeToInstances.get(failedImport.customObject.type).push(failedImport.customObject.id)
+      }
+    })
+    return Object.fromEntries(failedTypeToInstances.entries())
```

Comment:
Consider this implementation (not sure it is better)
```suggestion
    return _(importResult.failedImports)
      .filter(failedImport => {
        if (failedImport.customObject.result.message.includes('unexpected error')) {
          log.debug('Failed to fetch (%s) instance due to SDF unexpected error: (%s)', failedImport.customObject.type, failedImport.customObject.id)
          return true
        }
        return false
      })
      .groupBy(failedImport => failedImport.type)
      .mapValues(failedImports => failedImports.map(failedImport => failedImport.id))
```